docs/redis_authentication.md

```diff
@@ -0,0 +1,72 @@
+# Redis Authentication
+
+To avoid any unauthorized access to the Redis database resulting in an unintended behavior, arising from the usage of Redis' passwordless `default` user.
```

Comment:
Please check if the below sounds better.
```suggestion
To prevent unauthorized access to the Redis database and mitigate unintended behavior associated with the use of Redis' passwordless `default` user.
```